tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -221,21 +221,24 @@ TfLiteStatus AllocationInfoBuilder::GetOfflinePlannedOffsets(
 
 TfLiteStatus AllocationInfoBuilder::AddScratchBuffers(
     internal::ScratchBufferRequest* scratch_buffer_requests,
-    ScratchBufferHandle* scratch_buffer_handles) {
+    ScratchBufferHandle* scratch_buffer_handles, int subgraph_idx) {
   // Set up allocation info for buffers.
-  for (size_t i = tensor_count_; i < tensor_count_ + buffer_count_; ++i) {
+  for (size_t i = 0, buffers_added = 0; i < buffer_count_; ++i) {
```

Comment:
the buffer_count_ here is really total_buffer_count_ (total for all subgraphs), is that understanding correct?

Reply:
correct.